vcd/config_test.go

```diff
@@ -61,10 +61,11 @@ func init() {
 // Structure to get info from a config json file that the user specifies
 type TestConfig struct {
 	Provider struct {
-		User     string `json:"user"`
-		Password string `json:"password"`
-		Token    string `json:"token,omitempty"`
-		ApiToken string `json:"api_token,omitempty"`
+		User                string `json:"user"`
+		Password            string `json:"password"`
+		Token               string `json:"token,omitempty"`
+		ApiToken            string `json:"api_token,omitempty"`
+		ServiceAccountToken string `json:"service_account,omitempty"`
```

Comment:
```suggestion
		ServiceAccountTokenFile string `json:"service_account_token_file,omitempty"`
```
If we implement full support for service accounts, this name will have a different meaning

Reply:
updated